plugins/lua/logistics.lua

```diff
@@ -91,6 +111,8 @@ local function print_status()
     print(('Total items marked for melting: %5d'):format(global_stats.total_melt))
     print(('Total items marked for trading: %5d'):format(global_stats.total_trade))
     print(('Total items marked for dumping: %5d'):format(global_stats.total_dump))
+    print(('Total items marked forbidden: %7d'):format(global_stats.total_forbid))
+    print(('Total items unforbidden: %12d'):format(global_stats.total_unforbid))
```

Comment:
"unforbidden" sounds like an action. how about `Total items not marked forbidden`?

Reply:
I think it's still valuable, at least for completeness sake. It's the complement to the forbidden count